### ISSUE TITLE
valid_ibinspectable requires explicit types

### ISSUE THREAD

**Issue by Igor-Palaguta** [realm/SwiftLint#2748]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

`valid_ibinspectable` requires explicit types.

Swiftlint fires warning here:
`@IBInspectable public var isSimulatingBarButtonItem = false`

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint
```

`warning: Valid IBInspectable Violation: @IBInspectable should be applied to variables only, have its type explicit and be of a supported type (valid_ibinspectable)`

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.32.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
Via package file from releases assets

* Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? no
* Which Xcode version are you using (check `xcode-select -p`)? Version 10.2.1 (10E1001)

```swift
// This triggers a violation:
import UIKit

public final class CloseButton: UIButton {
    @IBInspectable public var isSimulatingBarButtonItem = false
}
```

**Comment by Igor-Palaguta** [comment]
Interface Builder requires to add type explicitly in other case this property is not displayed as custom attribute.

**Comment by akovalov** [comment]
Adding `public` like `@IBInspectable public var cornerRadius: CGFloat` or `public extension UIView {` solved warning for me
